builds/e2e/templates/longhaul-deploy-iotuap.yaml

```diff
@@ -0,0 +1,41 @@
+parameters:
+  release.label: ''
+  edgelet.artifact.name: ''
+  images.artifact.name: ''
+  container.registry: ''
+  container.registry.username: ''
+  container.registry.password: ''
+  iotHub.connectionString: ''
+  eventHub.connectionString: ''
+  snitch.build.number: ''
+  snitch.alert.url: ''
+  snitch.storage.account: ''
+  snitch.storage.masterKey: ''
+  loadGen.message.frequency: ''
+
+steps:
+  - task: CopyFiles@2
+    displayName: 'Copy Edgelet Artifact'
+    inputs:
+      SourceFolder: "$(Build.StagingDirectory)/${{ parameters['edgelet.artifact.name'] }}"
+      TargetFolder: "\\\\$(iotuap-x64-machine)\\c$\\Data\\e2etests\\artifacts\\${{ parameters['edgelet.artifact.name'] }}"
+      CleanTargetFolder: true
+  - task: CopyFiles@2
+    displayName: 'Copy Images Artifact'
+    inputs:
+      SourceFolder: "$(Build.StagingDirectory)/${{ parameters['images.artifact.name'] }}"
+      TargetFolder: "\\\\$(iotuap-x64-machine)\\c$\\Data\\e2etests\\artifacts\\${{ parameters['images.artifact.name'] }}"
+      CleanTargetFolder: true
+  - task: PowerShellOnTargetMachines@3
+    displayName: 'Run Long Haul Deployment'
```

Comment:
```suggestion
    displayName: 'Run Long Haul Deployment on $(iotuap-x64-machine)'
```

Reply:
please try this suggestion, if it is possible to use variable in displayName field.

Reply:
Good suggestion.
Done.